src/ts/csv.ts

```diff
@@ -53,11 +53,15 @@ export function parseCsvFile(csvPath: string): Promise<Claim[]> {
 }
 
 export function writeCsv(claims: Claim[]): Writable {
-  const accounts = Array.from(new Set(claims.map(({ account }) => account)));
+  const accounts = Array.from(
+    new Set(claims.map(({ account }) => utils.getAddress(account))),
+  );
 
   const claimsByAccount: Record<string, Claim[]> = {};
   for (const user of accounts) {
-    claimsByAccount[user] = claims.filter(({ account }) => account === user);
+    claimsByAccount[user] = claims.filter(
+      ({ account }) => account.toLowerCase() === user.toLowerCase(),
+    );
```

Comment:
Not really important, but I guess this is just equivalent to iterating only once over the claims and grouping in `claimsByAccount`, so it would be `O(n)` instead of `O(n*m)`, being `n` the number of claims, and `m` the number of users.

Reply:
Good point! Fixed in #60.

Reply:
Ohh, i appreciate u doing the FIX! i just said it as nitpick, this is only for CSV generation, so i was not very concerned. 

anyways, good one!